superdesk/default_settings.py

```diff
@@ -739,3 +740,8 @@ def local_to_utc_hour(hour):
 
 #: Whether to generate custom crops on image upload by default or not, if false means generate else don't generate
 NO_CUSTOM_CROPS = strtobool(env('NO_CUSTOM_CROPS', 'false'))
+
+#: Authorisation server
+
+#: how long an authorisation token is valid (in seconds)
+AUTH_SERVER_EXPIRATION_DELAY = env('AUTH_SERVER_EXPIRATION_TIME', 60 * 60 * 24)  # 1 day by default
```

Comment:
I think `AUTH_SERVER_SHARED_SECRET = env('AUTH_SERVER_SHARED_SECRET' , '')` is missed here

Reply:
I think I didn't put it here because I didn't wanted a default value, but I could check against empty string, so I'll update.